### PR TITLE
BZ1267419: changing layout of release notes for 3.1

### PR DESCRIPTION
**Pull request by bfallonf**
WIP, just to show the as discussed Release Notes layout for 3.1.

### CODE REVIEW

**Comment by bfallonf** [comment]
@vikram-redhat @adellape  Here's the release notes. I have done this:

- Organised the Rel Notes top-level "folder", so there's no longer a "What's New" section, but we have this and it's what we discussed. (I can put in a "v2 vs v3" section later.)
- Put in a lot of info, but not all. I also haven't edited the info that's in there, so don't bother reading it.
- Fixed the links throughout the rest of the docs.

Bonus response
- I need to put in the rest of the Bug Fixes

Alex, do you know if merging this will break the origin Rel notes? I have deleted the "What's New" section from the build file, and added the "Release Notes" section, but this may break everything. HOWEVER, Vikram says this is now pointing at the OSE 3.1 branch, so maybe it won't hit the Origin docs?

Also, Alex, do you know if there's a way we can find if the stuff that was _previously_ on tech preview is still on tech preview? There was quite a lot of that throughout the docs, and instead of taking that whole paras away, I just made it point to the 3.0 Rel Notes, figuring the info is in there somewhere (even though it's not, really, cos I deleted it in the organising...blah). you can see what I mean by looking at the files I changed in the other tab above.

**Comment by adellape** [comment]
@bfallonf Remember to squash, please.

File: dev_guide/persistent_volumes.adoc [hunk 1]
```diff
@@ -16,34 +16,13 @@ Storage is provisioned by an administrator by creating `*PersistentVolume*`
...
-link:../rest_api/kubernetes_v1.html#v1-rbdvolumesource[RADOS (Ceph)], are
-currently in
-link:../whats_new/ose_3_0_release_notes.html#technology-preview[Technology
-Preview]. The Administrator Guide provides instructions on provisioning an
-OpenShift cluster with
-link:../install_config/persistent_storage/persistent_storage_nfs.html[persistent storage using NFS].
+The Installation and Configuration Guide provides instructions on provisioning
+an OpenShift cluster with
+link:../install_config/persistent_storage/persistent_storage_nfs.html[persistent
+storage using NFS].
```
Comment (adellape) [inline dev_guide/persistent_volumes.adoc]: We don't need to specifically call out NFS anymore, since we have multiple admin topics on various volume plug-ins now.

File: install_config/install/prerequisites.adoc [hunk 2]
```diff
@@ -86,17 +86,10 @@ completing the initial OpenShift installation depending on your application
...
-link:../../whats_new/ose_3_0_release_notes.html#technology-preview[Technology
-Preview].
-endif::[]
-ifdef::openshift-origin[]
-The Administrator Guide provides instructions on provisioning an OpenShift
-cluster with link:../../install_config/persistent_storage/persistent_storage_nfs.html[persistent
+The Installation and Configuration Guide provides instructions on provisioning
+an OpenShift cluster with
+link:../../install_config/persistent_storage/persistent_storage_nfs.html[persistent
 storage using NFS].
```
Comment (adellape) [inline install_config/install/prerequisites.adoc]: Same here.

File: install_config/install/quick_install.adoc [hunk 3]
```diff
@@ -166,10 +166,6 @@ an unattended installation, the installation fails.
 must be set to *true* for the configuration file to be considered valid.
 <10> If set to *true*, OpenShift services are run in a container on this host
 instead of installed using RPM packages.
-ifdef::openshift-enterprise[]
-Containerized installations are in Technology Preview in OpenShift Enterprise
-3.1.
-endif::[]
```
Comment (adellape) [inline install_config/install/quick_install.adoc]: Why was this removed? Looks like it was in your "fix conflicts" commit. This needs to be back in.

File: using_images/db_images/mysql.adoc [hunk 4]
```diff
@@ -324,7 +324,7 @@ access the database via the service that was defined.
 ifdef::openshift-origin[]
 [IMPORTANT]
 ====
-Replication support provided by the MySQL image is experimental and should not
+Replication support provided by the MongoDB image is experimental and should not
```
Comment (adellape) [inline using_images/db_images/mysql.adoc]: Looks like another casualty of fixing conflicts. This is the MySQL image topic.

File: welcome/index.adoc [hunk 5]
```diff
@@ -13,7 +13,7 @@ provide information to set up and manage your OpenShift environment.
 [cols="2",frame="none",grid="cols"]
 |===
 
-a|link:../whats_new/index.html[*What's New*]
+a|link:../release_notes/index.html[*What's New*]
```
Comment (adellape) [inline welcome/index.adoc]: This doesn't exist in the Origin build.

**Comment by adellape** [comment]
@bfallonf I've addressed my comments in https://github.com/openshift/openshift-docs/pull/1183 as a follow-up.
